melodic/distribution.yaml

```diff
@@ -3905,6 +3905,23 @@ repositories:
       url: https://github.com/swri-robotics/gps_umd.git
       version: master
     status: developed
+  gpu-voxels-release:
```

Comment:
This name does not follow packaging naming rules ([REP-144](https://www.ros.org/reps/rep-0144.html)). We should use underscores (`_`) instead of dashes (`-`). Also, consider dropping the `-release` suffix, which doesn't add to the package name IMO.

Reply:
So, I'm not sure what our policy is on _repository_ names, but package names definitely need to use `_`.

And to clear up any confusion this is supposed to be the name of the source repository, but including the `-release` suffix makes it seem like you're confusing it with the release repository.

Either way, I'd recommend following @jacobperron's suggestion and using `gpu_voxels` here.

Reply:
Changing the name cause an error on validation. Can you help me by detecting why?

Reply:
I think you should re-bloom the package (with the updated name). I think you can do this with the `--edit` option when running `bloom-release`.

Reply:
That generated another pull-request #31305.